.github/workflows/posix.yml

```diff
@@ -202,7 +202,13 @@ jobs:
           --target test
 
     - name: Generate code coverage report
-      if: inputs.enable_coverage == true
```

Comment:
Did you mean for this to be included? I know that it is annoying to have these coverage tests fail, but it seemed orthogonal to the point of this patch.